handlers/ready.go

```diff
@@ -0,0 +1,93 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/openearthplatforminitiative/agriculture-api/config"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"sync"
+)
+
+func Ready(c *gin.Context) {
```

Comment:
I think I would prefer to change the way of doing the parallell calls to be more similar as in `summary.go`.
The ready function could possibly also return a detailed information about either the service is ready, or which of the backing apis are not ready.

That way a client could check the readiness-function, before doing a summary-call. E.g if a client needs weather, flood and deforestation, but not soil, a call to the summary function can still be done even if the ready-call fails on just soil.